packages/blockchain-wallet-v4/src/redux/data/sfox/sagaRegister.js

```diff
@@ -14,9 +14,9 @@ export default ({ api, options }) => {
     yield takeLatest(AT.FETCH_SFOX_ACCOUNTS, dataSfoxSagas.fetchSfoxAccounts)
```

Comment:
I don't think this is neccessary

Reply:
If I deleted this line, user account information wouldn't show in trade details, so I left it in.